.github/workflows/plugin_install.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     strategy:
       matrix:
```

Comment:
@scrawfor99  This is a good reason to have the strategy to `fail-fast: false`

Reply:
Will do. See above, but I am hoping that for now we can accept the changes as they stand. If you would be willing to re-review it would be appreciated. The only thing that should have changed is that now there is the fail-fast policy you mentioned.